invenio_records_ui/permissions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
Isn't this just pushing the problem from inveniosoftware/invenio-records#137 down the stack? I.e. I don't think it should be added here.

For the example app, i think you can create an example of a permission factory that relies on metadata inside the record.